### ISSUE TITLE
Exporting joint co-ordinates into CSV file for post processing ?

### ISSUE THREAD

**Issue by ns3new** [stefanopini/simple-HRNet#36]
Is it possible to export `keypoints` of the joints into a csv /excel file to do post processing ?

**Comment by stefanopini** [comment]
I've added the script extract-keypoints.py in the latest commit!
Please refer to README.md for further information

**Comment by ns3new** [comment]
Hi thanks for the information :  I tried to execute the program as you had suggested. It seems there is a problem in it. When i tried to save keypoints from a saved value , it fails to do so. I tried to run a print statement after 
https://github.com/stefanopini/simple-HRNet/blob/08c35047f5f6b7dbc9dd41fd4f16581fe433c9c6/scripts/extract-keypoints.py#L59

It only prints out the pts for one frame and exists the program . Perhaps i am doing something wrong 

Executred command is 
`python3.6 scripts/extract-keypoints.py --filename cut_video.mp4  --csv_output_filename test.csv
`
